client/package.json

```diff
@@ -41,11 +49,16 @@
         "eslint-plugin-react-compat": "0.0.3",
         "if-env": "^1.0.0",
         "ignore-styles": "^5.0.1",
+        "jest": "^22.4.2",
+        "jest-css-modules": "^1.1.0",
         "mocha": "^5.0.0",
         "preact-cli": "^2.0.1",
         "preact-cli-sw-precache": "^1.0.3",
+        "preact-compat-enzyme": "^0.2.5",
         "preact-jsx-chai": "^2.2.1",
-        "preact-material-components": "^1.3.5"
+        "preact-material-components": "^1.3.5",
```

Comment:
Not sure when this one snuck back in on us but we should be able to remove teh material-components package

Reply:
I was wondering about that - I just took it out.